### ISSUE TITLE
[Bug Report][2.2.3] v-select doesn't work with screen reader

### ISSUE THREAD

**Issue by kodeo** [vuetifyjs/vuetify#15088]
### Environment
**Vuetify Version:** 2.2.3
**Vue Version:** 2.6.11
**Browsers:** Google Chrome, Mozilla Firefox
**OS:** Windows

### Steps to reproduce
Tab onto the following element using Windows screen reader NVDA...

```
<v-select
  label="Delivery country"
  :items="countryOptions"
  v-model="deliveryCountry"
/>

data: () => ({
  countryOptions: ['UK', 'USA'],
  deliveryCountry: 'UK'
})
```

### Expected Behavior
The screen reader should read something similar to the equivalent `<select>`, which would be "*Delivery country combo box UK collapsed*" and on selection it should announce the new value e.g. "*Delivery country combo box USA collapsed*".

### Actual Behavior
The screen reader says "*Delivery country UK button collapsed submenu, delivery country edit readonly blank*", and on selection "*Selected, delivery country edit readonly blank, list, delivery country edit readonly blank*".

### Reproduction Link
<a href="https://codepen.io/kodeo/pen/QWQKpmx" target="_blank">https://codepen.io/kodeo/pen/QWQKpmx</a>

**Comment by KaelWD** [comment]
Duplicate of #15033
